bundles/org.openhab.binding.sensibo/src/main/java/org/openhab/binding/sensibo/internal/model/SensiboModel.java

```diff
@@ -45,8 +45,9 @@ public long getLastUpdated() {
     }
 
     public Optional<SensiboSky> findSensiboSkyByMacAddress(final String macAddress) {
-        final String macAddressWithoutColons = macAddress.replace(":", "");
-        return pods.stream().filter(pod -> macAddressWithoutColons.equals(pod.getMacAddress())).findFirst();
+        final String macAddressWithoutColons = macAddress.replace(":", "").toUpperCase();
+        return pods.stream().filter(pod -> macAddressWithoutColons.equals(pod.getMacAddress().toUpperCase()))
+                .findFirst();
```

Comment:
Could be simplified to:
```suggestion
        final String macAddressWithoutColons = macAddress.replace(":", "");
        return pods.stream().filter(pod -> macAddressWithoutColons.equalsIgnoreCase(pod.getMacAddress())).findFirst();
```